.github/workflows/test.yaml

```diff
@@ -11,9 +11,17 @@ jobs:
     runs-on: ubuntu-latest
     name: Test - OTP ${{matrix.otp}} / Elixir ${{matrix.elixir}}
     strategy:
+      # see https://hexdocs.pm/elixir/1.14.0-rc.0/compatibility-and-deprecations.html#compatibility-between-elixir-and-erlang-otp
       matrix:
-        elixir: [1.11.x, 1.12.x]
-        otp: [22.x, 23.x, 24.x]
+        elixir: ['1.11.x', '1.12.x', '1.13.x']
+        otp: ['22.x', '23.x', '24.x']
+        include:
+          - elixir: '1.10.x'
+            otp: '22.x'
+          - elixir: '>1.10.3'
+            otp: '23.x'
+          - elixir: '1.13.4'
```

Comment:
can be 1.13.x, no? I would only run on the latest patch versions... and otp 25 is supported _starting_ from 1.13.4, but all later patches are going to support otp 25.